python/ray/autoscaler/aws/config.py

```diff
@@ -115,6 +115,14 @@ def _configure_key_pair(config):
         assert "KeyName" in config["head_node"]
         assert "KeyName" in config["worker_nodes"]
         return config
+    elif "RAY_AWS_SSH_KEY" in os.environ:
```

Comment:
Why do we need this instead of just putting `ssh_private_key` and `KeyName` the config?

Reply:
It feels wrong to put a special case in our autoscaler to simplify running stress tests on Jenkins. That kind of logic should be handled outside of the autoscaler (e.g., in the stress testing script).